infrastructure/smart-contracts/contracts/RefereeCalculations.sol

```diff
@@ -21,32 +21,50 @@ contract RefereeCalculations is Initializable, AccessControlUpgradeable {
-     * 4. The function returns the challenge emission and the emission tier.
+     * 2. The challenge emission is the emission tier divided by the total number of seconds in a year.
+     * 3. Calculate the time difference between challengeStart and challengeEnd in seconds.
+     * 4. Multiply the emissions per second by the time passed to get the total emissions for the challenge.
+     * 5. Return the total emissions and the emission tier.
      *
      * @param totalSupply The current total supply of tokens.
      * @param maxSupply The maximum supply of tokens.
-     * @return uint256 The challenge emission.
-     * @return uint256 The emission tier.
+     * @param challengeStart The start time of the challenge in Unix timestamp format.
+     * @param challengeEnd The end time of the challenge in Unix timestamp format.
+     * @return uint256 The total challenge emission based on the time passed.
+     * @return uint256 The emission tier for the current challenge.
      */
     function calculateChallengeEmissionAndTier(
         uint256 totalSupply,
-        uint256 maxSupply
+        uint256 maxSupply,
+        uint256 challengeStart,
+        uint256 challengeEnd
     ) public pure returns (uint256, uint256) {
         require(maxSupply > totalSupply, "5");
+        require(challengeEnd >= challengeStart, "7"); // Ensure that end time is not before start time
 
-        uint256 tier = Math.log2(maxSupply / (maxSupply - totalSupply)); // calculate which tier we are in starting from 0
-        require(tier <= 23, "6");
+        // 1. Calculate the time difference in seconds
+        uint256 timePassedInSeconds = challengeEnd - challengeStart;
 
-        uint256 emissionTier = maxSupply / (2 ** (tier + 1)); // equal to the amount of tokens that are emitted during this tier
+        // 2. Determine the current emission tier using a logarithmic approach
+        // The tier is calculated as the logarithm base 2 of the proportion of total supply to the remaining supply
+        uint256 tier = Math.log2(maxSupply / (maxSupply - totalSupply));
+        require(tier <= 23, "6"); // Limiting tier to a maximum value
 
-        // determine what the size of the emission is based on each challenge having an estimated static length
-        return (emissionTier / 17520, emissionTier);
+        // 3. Calculate the emission tier, which is the maximum supply divided by 2^(tier + 1)
+        uint256 emissionTier = maxSupply / (2 ** (tier + 1)); // Equal to the amount of tokens emitted during this tier
+
+        // 4. Calculate the emission rate per second, based on the emission tier
+        uint256 emissionPerSecond = emissionTier / 31536000; // 31536000 is the number of seconds in a non-leap year
+
+        // 5. Calculate the total emissions based on the time passed
+        uint256 totalEmissions = emissionPerSecond * timePassedInSeconds;
+
+        // 6. Return the total emissions and the emission tier
+        return (totalEmissions, emissionTier);
```

Comment:
@XDapps Im not sure this is correct.

The original function was dividing by 17520 to get the ~ hourly reward amount.

17520 seems to be the number of hours in two years. If we don't keep this exactly the same we will not conserve the calcualtion as it was before right ?

Do you see any issues with just extending the function with the lines dividing the original `emissionTier / 17520` and then calcualte the amount / second from there ?

At least we will have to make sure if we call that function for a duration of 1 hour we will get the current reward amount that is being emitted.